docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -170,10 +170,32 @@ correct Google Workspace plan.
 
 ### Create an OAuth client ID
 
-In the [Create an OAuth client ID](https://console.cloud.google.com/apis/credentials/oauthclient) page of the GCP console, select "Web application" as the Application type, pick a name, then add the following as an authorized redirect URI.
+#### Determine your callback URL
+
+You'll need a callback URL in order to complete the creation of an OAuth client ID with Google. For
+most deployments, this URL is your `public_addr` value. This consists of a domain name and port number.
+
+Most deployments use port 443 or port 3080. The callback URL you use might look like the following examples (replace example.teleport.com or example.teleport.sh with your cluster's value):
+
+Teleport Cloud: `https://example.teleport.sh:443/v1/webapi/oidc/callback`
+
+Teleport Enterprise: `https://example.teleport.com:443/v1/webapi/oidc/callback`
```

Comment:
Yes, and I thought about making that edit too - I just thought it might be confusing for people to see no port vs 3080 on the next line. On the whole I would prefer to see no port as the browser is going to strip it from the URL the second it sees it anyway.

Reply:
Well, if the context here is determining what URL to provide to Google to configure them as an IdP, then the determining factor should be if the field this URL is going into accepts https with or without the port specified (when it's default `443`).

Reply:
@zmb3 I am fine with no port, but the current example specifies 3080 without any additional context. I had a customer get stuck on this for a long time because of the 3080 example.

Reply:
@zmb3 I just removed the callback URL section and added a note to Configure the OAuth Client ID section on what the callback URL should be.